### ISSUE TITLE
The app prompts to install the JDK instead of the JRE

### ISSUE THREAD

**Issue by aaronsturm** [TeamSQL/desktop-app#382]
In version 4.0.341, I was trying to connect to a SQL Server and it prompted me to install the JDK. At the bare minimum, this should be changed to JRE as only the runtime is required.

**Comment by JoelNietoTec** [comment]
I've installed both JDK and JRE and the app is still preventing me to use it. Is very frustating.

**Comment by aaronsturm** [comment]
@JoelNietoTec - In Ubuntu I ran `apt update && apt install -y default-jre` to get mine working. Hopefully that helps you.

**Comment by agentwolf44** [comment]
Yes, I'm extremely frustrated right now. Whenever I try to connect to SQL it keeps prompting me to download the jdk or jre or whatever it needs, and I downloaded and installed it and it still keeps showing me this stupid message.

**Comment by IKaftanov** [comment]
@agentwolf44 restarting the program fixed this problem for me.

**Comment by DineWithMe** [comment]
https://www.quora.com/How-do-I-know-whether-jdk-is-installed-in-my-computer-or-not

guys the solution is here. you need to set the environment variable path

**Comment by vishnu-dev** [comment]
I too experience the same issue
![Screenshot of the error](https://user-images.githubusercontent.com/21088161/47549193-f654df80-d918-11e8-83e7-1b0836a04511.png)
Have installed latest jdk

**Comment by jcklpe** [comment]
@aaronsturm Thank you! Your solution worked like a charm.

**Comment by Dottenpixel** [comment]
Having the same issue on a Mac, installed JDK and JRE, but problem persists.

**Comment by jeongsd** [comment]
`sudo apt install default-jdk` work for me
https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-ubuntu-18-04
